easycheck/easycheck.py

```diff
@@ -758,7 +758,7 @@ def check_argument(
             the default text 'argument'
         expected_type (type, Iterable[type]): the expected type of the item
         expected_choices (Iterable): a list of acceptable values of argument
-        expected_length (int): the expected length of the item
+        compare_length_to (int): the expected length of the item
```

Comment:
Please revise the description, in a similar way you did for `compare_to` in `check_length()`

Reply:
done